pkg/config/config.go

```diff
@@ -299,3 +316,16 @@ func (c *Config) GetAuthChecker() (auth.Checker, error) {
 	}
 	return auth.NewChecker(checkType, client.NewInCluster)
 }
+
+func (c *Frontend) GetAggregateKeyLabels() map[string][]string {
+	keyLabels := map[string][]string{
+		"app":          {"app"},
+		"droppedState": {"PktDropLatestState"},
+		"droppedCause": {"PktDropLatestDropCause"},
+		"dnsRCode":     {"DnsFlagsResponseCode"},
+	}
+	for _, sc := range c.Scopes {
+		keyLabels[sc.ID] = sc.Labels
+	}
```

Comment:
This should fix the linter warning
```suggestion
	for i := range c.Scopes {
		keyLabels[c.Scopes[i].ID] = sc.Labels
	}
```